cmd/osnadmin/main.go

```diff
@@ -52,7 +52,8 @@ func executeForArgs(args []string) (output string, exit int, err error) {
 	configBlockPath := join.Flag("config-block", "Path to the file containing an up-to-date config block for the channel").Short('b').Required().String()
 
 	list := channel.Command("list", "List channel information for an Ordering Service Node (OSN). If the channelID flag is set, more detailed information will be provided for that channel.")
-	listChannelID := list.Flag("channelID", "Channel ID").Short('c').String()
+	info := channel.Command("info", "Get detailed information about a specific channel.")
+	infoChannelID := info.Flag("channelID", "Channel ID").Short('c').Required().String()
```

Comment:
I'm thinking we can do this in a backward compatible way.
Keep the support for passing channel id to list with this line:
`listChannelID := list.Flag("channelID", "Channel ID").Short('c').String()`

Then add an entirely new "info" command.

Change the function osnadmin.ListSingleChannel() to osnadmin.Info() and have both of them call it.

The help text doesn't need to mention that you can pass channel id to list command anymore, but keep the functionality there so that any existing users that depend on it don't break.